### PR TITLE
Adding new post on 2fa.

### PR DESCRIPTION
**Pull request by rdegges**
For publication on FRIDAY!!! Don't merge until then.

### CODE REVIEW

File: _source/_data/authors.yml [hunk 1]
```diff
@@ -255,3 +255,11 @@ reverentgeek:
   web: https://reverentgeek.com
   twitter: https://twitter.com/reverentgeek
   github: https://github.com/reverentgeek
+
+cgilsenan:
+  full_name: Conor Gilsenan
+  display_name: Conor Gilsenan
+  avatar: avatar-cgilsenan.jpg
+  web: https://reverentgeek.com
```
Comment (mraible) [inline _source/_data/authors.yml]: This doesn't seem right.

File: _source/_posts/2018-09-14-how-to-get-more-users-to-enable-2fa-on-their-accounts.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+layout: blog_post
+title: 'How to Get More Internet Users to Enable 2FA on Their Accounts'
+author: cgilsenan
+description: "Interested in ways to improve two-factor authentication for your users? We'll teach you how to grow two-factor adoption and build a more secure userbase."
+tags: [security, mfa]
+tweets:
+ - "Want to build a more secure userbase? Get your users to enable two-factor authentication! Our buddy @conorgil wrote an awesome guide you might enjoy on the topic:"
+ - "We love two-factor! Learn how @conorgil is helping push for 2fa adoptoin in our latest post:"
```
Comment (mraible) [inline _source/_posts/2018-09-14-how-to-get-more-users-to-enable-2fa-on-their-accounts.md]: adoptoin > adoption

File: _source/_posts/2018-09-14-how-to-get-more-users-to-enable-2fa-on-their-accounts.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+If you are reading this article on the Okta Developer blog, chances are high that you are already quite familiar with two-factor authentication (2FA) and how it helps keep hackers out of user accounts even if they're using compromised passwords.
+
+You probably already have 2FA enabled on all of your online accounts.
+
+Sadly, you are in the significant minority.
+
+## Most People are Not Using 2FA
+
+Duo Security conducted a survey to research perceptions and adoption rates of 2FA among average internet users. The results, published in the State of the Auth, were sobering.
```
Comment (mraible) [inline _source/_posts/2018-09-14-how-to-get-more-users-to-enable-2fa-on-their-accounts.md]: link to "State of the Auth"?

File: _source/_posts/2018-09-14-how-to-get-more-users-to-enable-2fa-on-their-accounts.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+As useful as these resources are, the main drawback of both is that users need to go out of their way to *proactively* use them. The old saying “out of sight, out of mind” seems fitting here. I focus on 2FA professionally, and even I often forget to check those resources regularly to see if a new service I am signing up for supports 2FA. I think the average internet user is likely to forget too.
+
+## Enable Wider 2FA Adoption Passively
+
+Wouldn't it be awesome if we could educate users about which sites they visit support 2FA without them having to do anything?
+
+That is what [2FA Notifier](https://2fanotifier.org/) makes possible.
+
+2FA Notifier is an [open source web extension](https://github.com/conorgil/2fa-notifier) for Chrome and Firefox built by [yours truly](https://twitter.com/conorgil) and my partner in crime, [Ray Gonzales](https://ray-gonzales.com/), that notifies users anytime they visit a site that supports 2FA.
```
Comment (mraible) [inline _source/_posts/2018-09-14-how-to-get-more-users-to-enable-2fa-on-their-accounts.md]: I was going to suggest this extension after reading the above section. It's awesome that Conor created it!

File: _source/_posts/2018-09-14-how-to-get-more-users-to-enable-2fa-on-their-accounts.md (hunk 2 above)
Comment (rdegges) [reply _source/_posts/2018-09-14-how-to-get-more-users-to-enable-2fa-on-their-accounts.md]: Tyyyy

File: _source/_posts/2018-09-14-how-to-get-more-users-to-enable-2fa-on-their-accounts.md (hunk 3 above)
Comment (rdegges) [reply _source/_posts/2018-09-14-how-to-get-more-users-to-enable-2fa-on-their-accounts.md]: Can't, will message ya on slack.

**Pull request merged**